R/main.R

```diff
@@ -34,14 +34,19 @@ Options:
 
 main_worker_args <- function(args = commandArgs(TRUE)) {
   usage <- "Usage:
-orderly_worker [<queue_id>]"
+orderly_worker [options] [<queue_id>]
+
+Options:
+  --go-signal=PATH  Absolute path for go signal"
```

Comment:
Server line 54 implies this can be relative too (so just "path for go signal")

Reply:
Thanks, updated that